src/app/forms/components/select/select.component.html

```diff
@@ -1,18 +1,26 @@
-<div class="govuk-form-group">
-  <label class="govuk-label" for="{{ name }}"> {{ label }} </label>
-  <div id="{{ name }}-hint" class="govuk-hint">{{ hint }}</div>
+<div *ngIf="isEditing; else view" [ngClass]="{ 'govuk-form-group': !noBottomMargin }">
+  <h1 *ngIf="label" class="govuk-label-wrapper">
```

Comment:
Doesn't make sense semantically for there to be an `h1` here

Reply:
As discussed, this will be addressed as part of the bigger overhaul.